app/views/shared/_item_comment.html.erb

```diff
@@ -2,7 +2,7 @@
 <div class='form-group'>
   <%= f.label :item_comment, label_for_comments_field, class: "#{label_column_class} control-label required" %>
   <div class='<%= content_column_class %>'>
-    <%= f.text_area_without_bootstrap :item_comment, class: 'form-control', required: true %>
+    <%= f.text_area_without_bootstrap :item_comment, class: 'form-control', required: true, autocomplete: 'off' %>
```

Comment:
Same here, I wasn't aware of any issues w/ autocompletes in the textarea (and I'm not quite sure that they apply to them).